### DISCUSSION TITLE
Having difficulty editing keyboard shortcuts

### DISCUSSION THREAD

**Discussion by ArnoldBrody** [Stellarium/stellarium#1958] [Q&A]
In Stellarium 0.21.2, I would like to create a shortcut for the Action "Today's Evening Twilight". Currently, the Keyboard Shortcuts window shows no Primary shortcut or Alternative shortcut for this Action. I'd like [Ctrl] + {Alt] + T to be the shortcut for Today's Evening Twilight. 
  In the Keyboard Shortcuts Window (F7), I select the Action "Today's Evening Twilight" so It's highlighted. I tried pressing the keys [Ctrl]+[Alt]+T simultaneously thinking Stellarium would understand that's what I would like the shortcut to be, but nothing happens (the Primary shortcut field remains empty for this Action).
  If I double-click "Today's evening twilight", my cursor starts blinking in the Primary shortcut field found under "Search the action...". Unfortunately, I don't want to Search for an action. I want to add an action.
  With my cursor in Primary shortcut, I am not able to enter Ctrl for the Ctrl-key. If I press the Ctrl-key, nothing happens. If I type C (to start spelling Ctrl), the Keyboard Shortcuts jumps immediately to Constellations for which the primary shortcut is C. Then, when I type "t" for the second letter in Ctrl, Stellarium adds a comma after the C I typed first, then a space, and then the "t". In other words, you cannot spell Crtl, so I have no idea how you could search for a shortcut starting with a special character like Crtl, Alt, or Shift.
  My thanks in advance to anyone who can show me how to add a new shortcut!
Arnold

**Comment by gzotti** [comment]
You really press the Ctrl key, not type its name. 
But the problem may be that Ctrl-Alt-T is the default (preconfigured) key for the equation of Time plugin. You would have to either disable that plugin, or at least disable its shortcut first, before setting the new one. Else you just toggle Equation of time display instead of assigning that new key.

**Comment by axd1967** [comment]
@ArnoldBrody there is a workaround for this, see the related issue #1294 .

**Comment by ArnoldBrody** [comment]
Thank you, Georg.

**Comment by ArnoldBrody** [comment]
Thank you, Alex. I'll give that a try.
